README.md

```diff
@@ -0,0 +1,5 @@
+# openhardwaremonitor
+Open Hardware Monitor - a tool for monitoring hardware performance. Includes support for various temperature sensors, disk I/O ratings and power consumption.
+
+## forked from hexagon-oss/openhardware monitor
```

Comment:
This comment, as-is, is confusing when the PR is merged, so please remove it or change it.